client/ayon_blender/api/pipeline.py

```diff
@@ -155,6 +159,8 @@ def update_context_data(self, data: dict, changes: dict):
             changes (dict): Only data that has been changed. Each value has
                 tuple with '(<old>, <new>)' value.
         """
+        # Store in both properties for backward compatibility during transition
```

Comment:
Can we mark in the comment at what time we think this should be removed? Like, say in three months from now? Or six? or version X or Y?